torch/onnx/symbolic_opset11.py

```diff
@@ -893,9 +893,12 @@ def embedding_bag(g,
                   mode,
                   sparse,
                   per_sample_weights,
-                  include_last_offset):
+                  include_last_offset,
+                  padding_idx):
     if scale_grad_by_freq and sym_help._training_mode:
         return sym_help._onnx_unsupported('embedding_bag with scale_grad_by_freq for training mode')
+    if padding_idx is not None and padding_idx >= 0:
+        raise RuntimeError('embedding_bag with padding_idx')
```

Comment:
IIRC the onnx fallback operators are generated from the `native_functions.yaml` schema, so is it possible this would just work if you added `padding_idx`to the `g.op` call below?

Reply:
For opset9, yes, it does seem to be as simple as adding `padding_idx` to the `g.op("ATen", ..., operator_s="embedding_bag", ...)` call. But it looks like opset10 and opset11 are more complicated--the embedding_bag operation appears to be reimplemented with multiple `g.op` calls and I don't understand those yet. I'll try to figure it out

Reply:
After a little bit of investigation, I still don't really understand what has to be done here or how/if tests are being run for opset10 and opset11. Is there documentation that explains any of this? cc @mruberry

Reply:
Unfortunately I don't think there is. cc @bzinodev, @spandantiwari, who would know better

Reply:
Note: One common approach in these situations is to update the non-ONNX code and then file an issue explaining the discrepancy with ONNX.